### PR TITLE
Remove Button the Check And Fix Attributes

### LINKED ISSUE #2796: Buscador de Productos
- [x] Eliminar el botón azul de Ok.
imagen

- [x] Al darle doble clic para ver el detalle la `Existencia` sale en `NaN`.

- [ ] solop-develop/frontend-core#176

#### Other relevant information
- Your OS:
- Web Browser:
- Node.js version:
- adempiere-vue version:

### PR DESCRIPTION
**Pull request by elsiosanchez**
#### Screenshot or Gif

https://github.com/user-attachments/assets/b4fc5fc1-9dc4-4f67-90ee-0ad610decaf4

#### Additional context
fixes https://github.com/solop-develop/frontend-core/issues/2796

### CODE REVIEW

File: src/components/ADempiere/Form/ProductSearch/dialogo/infoProduct.vue [hunk 1]
```diff
@@ -126,6 +126,7 @@ export default defineComponent({
      */
 
     const product = computed(() => {
+      console.log({ product: store.getters.getCurrentProduct })
```
Comment (EdwinBetanc0urt) [inline src/components/ADempiere/Form/ProductSearch/dialogo/infoProduct.vue]: please remove this `console.log`

**Review by EdwinBetanc0urt — CHANGES_REQUESTED** [review]

**Comment by elsiosanchez** [comment]
Support to  Download in XLS
https://github.com/user-attachments/assets/e218bc5f-8b50-47bb-9e5f-b0f555b6e40b

File: src/components/ADempiere/Form/ProductSearch/dialogo/infoProduct.vue (hunk 1 above)
Comment (elsiosanchez) [reply src/components/ADempiere/Form/ProductSearch/dialogo/infoProduct.vue]: https://github.com/solop-develop/frontend-core/pull/2770/commits/9dec40da52f4127d13efafc64c0ff4e3901624dc

**Review by EdwinBetanc0urt — APPROVED** [review]

**Comment by Ricargame** [comment]
ref: https://github.com/solop-develop/frontend-core/issues/2796

**Pull request merged**